ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/ParticipantPruningIT.scala

```diff
@@ -49,6 +48,64 @@ class ParticipantPruningIT extends LedgerTestSuite {
     }
   })
 
+  test(
+    "PRQueryLatestPrunedOffsets",
+    "It should be possible to query the latest pruned offsets",
+    allocate(SingleParty),
+  )(implicit ec => { case Participants(Participant(participant, alice)) =>
+    for {
+      (prunedUpToInclusive_initial, allDivulgencePrunedUpToInclusive_initial) <-
+        participant.latestPrunedOffsets()
+      offsets <- populateLedgerAndGetOffsets(participant, alice)
+      offsetToPruneUpTo = offsets(lastItemToPruneIndex)
+
+      // Prune the ledger without divulgence
```

Comment:
I would suggest creating an example where we first pruneUpTo N without divulgence and then pruneUpTo N-1 with divulgence. Like this we could observe the independent move of both offsets

Reply:
This is what the test does:
* advance the non-divulgence pruning offset and assert that only it advanced
* advance the pruning with divulgence offset to the same offset and assert that only the latter advanced.

Note that they don't actually move independent: all-divulgence pruning includes normal pruning (so it advances both offsets).

Will add one more assertion below where both pruning offsets advance in tandem (all-divulgence pruning).